client/src/components/DescriptionComponent.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
this could be stateless as well
export default DescriptionInfo = ({ overviewInfo }) => (<StyledDescription>{overviewInfo.product_description})</StyledDescription>)}

Reply:
@danielharmon Good point! I am going to make these changes. Sometimes when I am moving too quickly i just make things class based, but this doesnt need to be.